lib/version-check.js

```diff
@@ -78,7 +80,7 @@ module.exports = function (gulp, gutil) {
         }
     }
 
-    function parseResponse(body, errorCallback) {
+    function parseResponse(body, opts, errorCallback) {
```

Comment:
Doesn't look like you're using opts in this function, is this in preparation for a future change?